gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/BulkControllerSupport.groovy

```diff
@@ -141,4 +149,18 @@ class BulkControllerSupport<D> {
         RepoLookup.findRepo(getEntityClass())
     }
 
+    /**
+     * Special handler for bulk operations, so that we can log/highight every bulk error we send.
+     * Its here, because we cant have more thn one exception handler for "Exception" in controller
+     */
+    Problem handleBulkOperationException(Exception e) {
+        assert getEntityClass()
+        Problem apiError = problemHandler.handleException(getEntityClass(), e)
+        if (apiError.status.code == 500) {
```

Comment:
So this is going to doubly log the stack trace right? once in handleException and again here? so I think we just need to logging before the handleException right?
again per the issue, see what was done for the getParams. we want to log out the request info so we can see and match it with the url and its obvious.

Reply:
@basejump it will not log error twice, because `handleException` wont get called in this case. 
We are catching the exception for bulk operation and explicitely handle it through `handleBulkOperationException`

I updated to log request info (uri, method, query string)

Reply:
problemHandler.handleException is called right above. its does a log.error in there.